### PR TITLE
improve alter edge and add alter edge UT

### PR DESCRIPTION
**Pull request by ayyt**
#349 Implement ALTER EDGE according to ALTER TAG
1. enhanced alter edge syntax
2. support alter edge sql

### CODE REVIEW

File: src/graph/AlterEdgeExecutor.cpp [hunk 1]
```diff
@@ -22,8 +22,61 @@ Status AlterEdgeExecutor::prepare() {
...
+    for (auto& edgeOpt : edgeOpts) {
+        nebula::meta::cpp2::AlterSchemaItem edgeItem;
+        auto opType = getEdgeOpType(edgeOpt->getOptType());
+        edgeItem.set_op(std::move(opType));
+        const auto& specs = edgeOpt->columnSpecs();
+        nebula::cpp2::Schema schema;
+        for (auto& spec : specs) {
+            nebula::cpp2::ColumnDef column;
+            column.name = *spec->name();
+            column.type.type = columnTypeToSupportedType(spec->type());
```
[the remark this answers is not in the document]
Comment (steppenwolfyuetong) [reply src/graph/AlterEdgeExecutor.cpp]: Yeah，TTL function will add separately in new PR. 
So ignore ttl here.

File: src/parser/MaintainSentences.h [hunk 2]
```diff
@@ -203,11 +203,67 @@ class AlterTagSentence final : public Sentence {
 };
 
 
+class AlterEdgeOptItem final {
```
[the remark this answers is not in the document]
Comment (dangleptr) [reply src/parser/MaintainSentences.h]: Agreed with @laura-ding .  We have unified them in thrift files.

[the remark this answers is not in the document]
Comment (steppenwolfyuetong) [reply src/parser/MaintainSentences.h]: 👍， I  will fix it.

**Comment by steppenwolfyuetong** [comment]
close this pr, use PR #394

**Pull request closed without merging**